src/gateways/createBookConfigLoader.ts

```diff
@@ -0,0 +1,48 @@
+import { BOOKS } from '../config';
+import { acceptStatus } from '../helpers/fetch';
+import Sentry from '../helpers/Sentry';
+
+export interface BookVersion {
+  defaultVersion: string;
+}
+
+interface BookConfig {
+  [key: string]: BookVersion;
+}
+
+interface ReleaseJsonStructure {
+  books: BookConfig;
+  code: string;
+  id: string;
+}
+
+const cachedBooks = { ...BOOKS } as BookConfig;
+
+export default (baseUrl: string) => {
+  const loadRemoteBookConfig = (): Promise<BookConfig | undefined> => {
+    const url = `${baseUrl}/rex/release.json`;
+    return fetch(url)
+      .then(acceptStatus(200, (status, message) => `Error response from "${url}" ${status}: ${message}`))
+      .then((response) => response.json() as Promise<ReleaseJsonStructure>)
+      .then((response) => response && response.books)
+      .catch((e) => {
+        Sentry.captureException(e);
```

Comment:
i can't imagine this being anything but a network failure, which will just be noise in sentry, worth reporting it but maybe make it a warning